### PR TITLE
raids: Adds title of "Re-loadable Layout" to raid scout overlay

### LINKED ISSUE #1252: DC Scout Notifier
Add an infobox on screen when a raid can be disconnect scouted

Currently known layouts: SCPFCCSPCF, CSPFCCCSSF, SCFPCSCPCF, PCSFCPCSCF, SCCFCPSCSF, SCPFCCCSSF, SCPFCPCSCF (courtesy of Maxed Sensei#0699)

### PR DESCRIPTION
**Pull request by ReeeMan**
in future someone might want to highlight chunks player has to stand in

https://i.imgur.com/DskZVYu.png - says "Re-loadable Layout" now

Open for suggestions to changing it

Closes #1252

### CODE REVIEW

**Comment by Ganom** [comment]
I'd like it to be a little bit more clear, people first coming into raids would see DC Scout and get confused. Maybe, "Re-loadable Layout"?. Can anyone else think of something?

File: runelite-client/src/main/java/net/runelite/client/plugins/raids/RaidsPlugin.java [hunk 1]
```diff
@@ -220,6 +220,11 @@
 	@Getter
 	private final List<String> layoutWhitelist = new ArrayList<>();
 
+	@Getter
+	private final ImmutableSet<String> list_of_DC_SCOUT_RAIDS = ImmutableSet.of(
```
Comment (Ganom) [inline runelite-client/src/main/java/net/runelite/client/plugins/raids/RaidsPlugin.java]: private static final, and it should be located in raidsoverlay, as it's not needed in the plugin.

**Comment by Kyleeld** [comment]
> I'd like it to be a little bit more clear, people first coming into raids would see DC Scout and get confused. Maybe, "Re-loadable Layout"?. Can anyone else think of something?

Perhaps “scouted layout”?

**Comment by ReeeMan** [comment]
I'll have a play around, could add the instance reload icon (that you see on the toolbar) to the title so people put 2 and 2 together, might see about adding an extra info box with a sentence about what a dc scout is.

**Comment by ReeeMan** [comment]
did i update this correctly or am i brainlet ?

**Comment by Ganom** [comment]
> 
> 
> did i update this correctly or am i brainlet ?

It would appear you did not :S

**Comment by ReeeMan** [comment]
> > did i update this correctly or am i brainlet ?
> 
> It would appear you did not :S

so brainlet.

I fixed it now

**Comment by xKylee** [comment]
@ReeeMan can you re-base this please?

**Pull request closed without merging**